Analysis/Core/include/Analysis/HFConfigurables.h

```diff
@@ -0,0 +1,186 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+// HF Configurable Classes
+//
+// Authors: Nima Zardoshti
+
+#ifndef O2_ANALYSIS_HFCONFIGURABLES_H
+#define O2_ANALYSIS_HFCONFIGURABLES_H
+
+#include <TMath.h>
+
+class HFTrackIndexSkimsCreatorConfigs
+{
+ public:
+  HFTrackIndexSkimsCreatorConfigs() = default;
+  ~HFTrackIndexSkimsCreatorConfigs() = default;
+
+  // 2-prong cuts D0
+  double mPtD0Min = 0.;
+  double mInvMassD0Min = 1.46;
+  double mInvMassD0Max = 2.26;
+  double mCPAD0Min = 0.75;
+  double mImpParProductD0Max = -0.00005;
+  // 2-prong cuts Jpsi
+  double mPtJpsiMin = 0.;
+  double mInvMassJpsiMin = 2.75;
+  double mInvMassJpsiMax = 3.45;
```

Comment:
The jpsi default upper mass can be lower, something like 3.20, but it can be changed later.
The default value for the minimum mass would probably be safer to have it lower as well. Thats because of the radiative jpsi tail and also because we normally use ee pairs with lower masses to estimate decay length for background pairs. So here i would suggest 2.50 as default

Reply:
hi @iarsene, thanks for the useful comment. I was wondering if at this point we should also increase the upper limit to include the psi(2S) peak, meaning get to 4.1 or so. I would say it should not substantially increase the size. It could be useful to measure that psi(2S) channel, but also to subtract it from the J/psi background shape. Let us know what you think. Cheers GM

Reply:
thanks @iarsene. we will include this in the next fix. cheers GM

Reply:
Thank you for the information, this has been updated